### ISSUE TITLE
Some hint characters  can not open.

### ISSUE THREAD

**Issue by Kzinux** [ueokande/vim-vixen#182]
### I'm opening this issue because:

- [x] I'll report a bug
- [ ] I'll propose a new feature

### Description
Hint begin with same character make the one character hint unusable.
### Failure Information (for bugs)
Can not open the hint.
#### Steps to Reproduce

1. Open www.qq.com.
2. Press f start follow.
3. Press a, hint 'a' doesn't open.

#### System configuration

- Operating system: Arch Linux x64
- Firefox version: Firefox 57
- Vim-Vixen version: Vim Vixen 0.5

#### Console logs

Any relevant log in developer tools:

**Comment by ueokande** [comment]
Press <kbd>Enter</kbd>

**Comment by Kzinux** [comment]
That works. 
Before firefox 57 i was using keysnail. When there are hints begin with same character,  keysnail doesn't use that character but two characters for hint so we don't need to press Enter. I think this is more comfortable.

**Comment by mathstuf** [comment]
It seems that the follow logic considers the empty string a "key" when generating follow strings. I use 5 characters for follow (to keep it to a one-hand home row) and I've been seeing follow sets like `AF`, `AFA`, `AFS`, `AFD`, `AFF`, and `AFG` together. I think it'd be better to just skip the follow hints which are prefixes of others. Note that uzbl is GPLv3, so copying the code directly isn't recommended. Looking at the hint key producer here, it seems to use the empty string as the "0" rather than the first character in the hintkey list.

**Comment by massey101** [comment]
I agree although this probably puts it in the new feature territory. It means that you can't be sure what sequence of keys to press beforehand thus slowing you down. If you don't know that AFA exists elsewhere on the page you would expect pressing AF would work, but the true sequence would be AF<enter>. Propose either AF shouldn't exist as an option or it should be displayed as AF<enter> or something similar.

**Comment by ouuan** [comment]
One more reason to skip one-character hints when out of use: While I'm looking at the one-character hint (for example, `A`), it's hard for me to notice the change that all hints except the ones that start with `A` disappeared, so I don't receive any response by pressing `A`. Thus, it makes me wonder whether I have pressed `A` or not.
